fms2_io/include/array_utils.inc

```diff
@@ -23,10 +23,11 @@
 !> @{
 
 !> @brief Allocate arrays using an input array of sizes.
-subroutine allocate_array_i4_kind_1d(buf, sizes)
+subroutine allocate_array_i4_kind_1d(buf, sizes, initialize)
```

Comment:
Are the changes in this file supposed to be here? I don't see the added arg used anywhere.

Reply:
I had originally thought that all modules within the same interface needed the same number of arguments but I was mistaken.  So this will all be removed in my next commit.